setup.py

```diff
@@ -50,6 +50,7 @@
         'collective.indexing',
         'plone.app.contentlisting',
         'plone.app.layout',
+        'plone.namedfile[blobs]',
```

Comment:
The blobs extra is deprecated a long time ago and will be removed in the next major release.
https://github.com/plone/plone.namedfile/commit/69fb864058fb1152ad813b651657c5e06d0c9f00

Reply:
Yes, but it's still present as of the current version. I'm all for making this a soft dependency soon, but as of know it's an undeclared, hard dependency (`plone.namedfile`, maybe not so much the `[blobs]` extra).